tests/migration/migration.go

```diff
@@ -1697,7 +1697,7 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 					libnet.WithMasqueradeNetworking(),
 					libvmi.WithResourceMemory("1Gi"),
 					// this annotation causes virt launcher to immediately fail a migration
```

Comment:
Not sure here, but the comment seems to be outdated - shouldn't we rely on what is present on the annotation documentation anyway?

Reply:
Done! Thanks

Reply:
For completeness, the issue was introduced here https://github.com/kubevirt/kubevirt/pull/12838/files#diff-5034f7c82b377dd518dd147554ad7df560b0cc667f1f0aa08e7a632d1436237aR1704:
![image](https://github.com/user-attachments/assets/f3ee143b-4d1c-48a5-ae95-405fb7a6f74a)